odoo/openupgrade/doc/source/modules100-110.rst

```diff
@@ -1,5 +1,5 @@
-Module coverage 10.0 -> 11.0
-============================
+Module coverage 9.0 -> 10.0
```

Comment:
Why this change?

Reply:
ah this was from the rebase i thought i changed it back